helixbass/tsc-rust
# Type-check string literal

In this PR:
- type-check a variable declaration of a string literal type

To test:
If you run `cargo run tmp.tsx` against a source file containing eg `const a: 'foo' = 'baz'` you should see a diagnostic debug-logged with message `Type '"baz"' is not assignable to type '"foo"'.`
If you run `cargo run tmp.tsx` against a source file containing eg `const a: 'foo' = 'foo'` you shouldn't see any diagnostics debug-logged

Based on `parse-string-literal`
